nifti2/nifti_tool.c

```diff
@@ -6508,8 +6507,11 @@ int diff_hdr1s( nifti_1_header * s0, nifti_1_header * s1, int display )
    for( c = 0; c < NT_HDR1_NUM_FIELDS; c++, fp++ )
       if( diff_field(fp, s0, s1, 1) )
       {
-         if( display ) disp_field(NULL, fp, s0, 1, ndiff == 0);
-         if( display ) disp_field(NULL, fp, s1, 1, 0);
+         if( display )
```

Comment:
Such changes are unnecessary.

Reply:
This will have been:

`nifti2/nifti_tool.c:6512:14: warning: The if condition is the same as the previous if condition [duplicateCondition]`

Indeed there was no bug in this case, but the warning is generally very useful, I've seen it catch copy-paste errors, and errors where the equality of conditions is not obvious to the human.

Reply:
Here's a [random example](https://gitlab.com/gromacs/gromacs/-/issues/4812) (first page search result) of the kind of copy-paste bugs this warning finds.